BuildDetails.md

```diff
@@ -0,0 +1,39 @@
+This is the build description guide that explains the build process for all FullScreenShenanigans projects. The guide will explain to you how the project is built and why it is built that way.
+
+### Table of Contents
```

Comment:
In general, most of the explanations are too short. Try to read this through the lens of someone first starting to look at the projects. What information would be useful to you? What wouldn't be? What would confuse you?